### ISSUE TITLE
Trojan znyonm in latest (1.0.0.1)

### ISSUE THREAD

**Issue by qedisme** [NatroTeam/NatroMacro#457]
### Have you searched the existing bug reports?

Yes

### What happened?

Windows AV found the Trojan. AV DB was updated 3 hours ago.
![trojan-in-natro](https://github.com/NatroTeam/NatroMacro/assets/7192836/59c490de-3b5a-41b5-8a42-712f4d753117)

### Steps to reproduce

1. Download
2. Scan
3. Virus

### What would you expect to happen?

Infection. And a complete lack of trust

### Natro Macro version

1.0.0.1

**Comment by ssilz** [comment]
Could you possibly have downloaded the macro from a different place? I've personally never experienced natro being detected as a possible trojan/virus.

**Comment by qedisme** [comment]
@ssilz No, I downloaded from the `latest release` link in this repo. `https://github.com/NatroTeam/NatroMacro/releases/latest`

**Comment by ninjubaer** [comment]
@qedisme if you downloaded it from the [official natromacro repo](https://github.com/natroteam/natromacro) then I can ensure you that you have no virus, at least not from natro macro. There are some AntiVirusses that detect AutoHotkey (.ahk) scripts and Batch (.bat) files as dangerous because they can be. Natro Macro isn't

**Comment by qedisme** [comment]
@ninjubaer Your reply is basically saying "trust me bro", and throwing in "...at least not from natro macro." is even less encouraging. 

I'm not expecting _any_ level of support. I was simply reporting this to your project in good faith. 

Your reply tells me all I needed to know.
